### PR TITLE
refactor: prepare react 18 upgrade - batch #16

### PR DESCRIPTION
**Pull request by neatbyte-vnobis**
## Changes
Sixteenth batch of changes to React component definitions before upgrading to React 18.
https://www.notion.so/webiny/Upgrade-to-React-18-ceacd9bbfcf74b39a7ceb70943c3afa7

Packages: app-security, app-theme, app-tenant, app-tenant-manager, app-security-access-managment

## How Has This Been Tested?
Manually.

## Documentation
None.

### CODE REVIEW

File: packages/app-security/src/Security.tsx [hunk 1]
```diff
@@ -5,8 +5,12 @@ import React from "react";
...
 import { SecurityProvider as ContextProvider } from "./contexts/Security";
 
-const SecurityProviderHOC = (Component: React.FC<any>): React.FC<any> => {
-    return function SecurityProvider({ children }) {
+interface SecurityProviderProps {
+    children: React.ReactNode;
+}
+
+const SecurityProviderHOC = (Component: React.ComponentType) => {
+    return function SecurityProvider({ children }: SecurityProviderProps) {
```
Comment (neatbyte-ivelychko) [inline packages/app-security/src/Security.tsx:13]: Is this okay that I have started changing types for `Components` as props from `FC` to `ComponentType`?

Comment (neatbyte-ivelychko) [reply packages/app-security/src/Security.tsx:13]: @brunozoric

Comment (brunozoric) [reply packages/app-security/src/Security.tsx:13]: Ok, for now. We will see if there will be issues with React 18.

**Review by brunozoric — APPROVED** [review]

**Pull request merged**